api.go

```diff
@@ -32,21 +32,17 @@ var (
 	// ErrEnqueueTimeout is returned when a command fails due to a timeout.
 	ErrEnqueueTimeout = errors.New("timed out enqueuing operation")
 
-	// ErrKnownPeer is returned when trying to add a peer to the configuration
```

Comment:
Realized we nuked these in a way that Consul needed. It wasn't a big deal to recover by checking the configuration beforehand, which is probably cleaner than looking for a specific error. Deleting these so people know this interface is gone.